test-cases/longevity/longevity-large-partition-2days.yaml

```diff
@@ -31,9 +31,13 @@ user_prefix: 'longevity-large-partitions-2d-order-by-desc'
 
 # To validate rows in partitions: collect data about partitions and their rows amount
 # before and after running nemesis and compare it
-validate_partitions: true
-table_name: "scylla_bench.test"
-primary_key_column: "pk"
+data_validation: |
+  validate_partitions: true
+  table_name: "scylla_bench.test"
+  primary_key_column: "pk"
+  max_partitions_in_test_table: 1000
+  partition_range_with_data_validation: 0-800
 
 
 run_fullscan: ['{"mode": "random", "ks_cf": "scylla_bench.test", "interval": 3600, "pk_name":"pk", "rows_count": 50000, "validate_data": true}']
+nemesis_selector: ["delete_rows", "disruptive"]
```

Comment:
This is going to change this test completely, it's ok to have it and if it's an exact dup of the 4days one, let's just rename it to something more meaningful like "longevity-large-partitions-deletions-and-disruptive-only" (just an example).